setup.py

```diff
@@ -89,6 +89,7 @@ def is_requirement(line):
     },
     include_package_data=True,
     install_requires=load_requirements('requirements/base.in'),
+    extras_require={"django": ["Django>=2.2,<2.3"]},
```

Comment:
Slightly cleaner would be to use something like `extras_require={"django": load_requirements('requirements/django.in')},` here, update `Makefile` to create `django.txt` from the new input file, and include `-r django.txt` in `requirements/test.in`.  `requirements/django.in` would just contain an explanatory comment and the `Django>=2.2,<2.3` constraint you have here.  This would remove keep Django out of `base.in`, `base.txt`, and the core package dependencies while keeping the rest of the requirements files for testing and local development largely unchanged.

Reply:
Indeed, much better.

Reply:
Updated the requirement files as per suggestion.